VT Update 1

VT-14 opened the pull request:
Please let me know what you think of this format and writing style.

Added content to the following pages based on the 1.10 version of the mod:
* Exit Rail
* Stirling Generator
* Combustion Generator
* Zombie Generator
* Alloy Smelter
* Photovoltaic Cells
* Capacitor Banks
* SAG Mill
* Power Monitors

Rearranged and added a little more to the Sidebar.

Renamed the Capacitor page to Capacitors.

VT-14 committed:
VT Update 1

Please let me know what you think of this format and writing style.

Added content to the following pages based on the 1.10 version of the mod:
* Exit Rail
* Stirling Generator
* Combustion Generator
* Zombie Generator
* Alloy Smelter
* Photovoltaic Cells
* Capacitor Banks
* SAG Mill
* Power Monitors

Rearranged and added a little more to the Sidebar.

Renamed the Capacitor page to Capacitors.

HenryLoenwind commented on SAG-Mill.md:
```diff
@@ -1,2 +1,26 @@
+Crushes items.  Used for ore doubling and as a crafting process.
+
+The slot on the right side of the GUI is for SAG Mill Grinding Balls.  These additives increase the yield and reduce the power required for certain recipes.  The Grinding Balls are consumed, but do last for several operations.  The Main Output is a bonus to all of the possible outcomes.  The Bonus Output is applied multiplicatively to any secondary outputs.  See the example at the bottom of the p ...
+
```
Can you reword the ball text a bit? This is what I put into the recipe xml file:

```
 Grinding Balls:
  Grinding balls provide bonus outputs and reduced power use. Using the user config new grinding balls may be added and
  existing ones removed.
  
  Use of the grinding ball is restricted by the bonusType of the recipe. If it is set to "none", it will not have any grinding
  ball bonuses applied and will not reduce the durability of any balls in the mill at the time it is  processed. If it is set
  to "chance_only", the chance and power multipliers of the grinding balls are applied, but not the grinding multiplier. If it
  is set to "multiply_output" (default), all multipliers apply.
  
  - grindingMultiplier: The recipe output will be produced this many times. e.g. "3.14" would produce 3 copies of the complete
                        output set and a 14% chance for a forth copy.
  - chanceMultiplier:   The chances for the different outputs of the recipe are multiplied with this. e.g. if a recipe specifies
                        <itemStack modID="minecraft" itemName="cobblestone" chance="0.15" /> a chanceMultiplier of "3.14" would
                        make it a 47.1% chance of getting one cobblestone. Results of more than 100% do not give multiples.
  - powerMultiplier:    The "energyCost" of the recipe is multiplied with this.
  - durationRF:         This specifies how long one grinding ball will last before it is used up. The unit is the amount of
                        power used by the SAG Mill.
```

(grindingMultiplier="main" chanceMultiplier="bonus" in the GUI)

There is no "main" or "secondary" output, there only is "output with 100% chance that isn't impacted by the chanceMultiplier because of that" and "output with less than 100% chance"

HenryLoenwind commented on SAG-Mill.md:
```diff
@@ -1,2 +1,26 @@
+
+Without a Grinding Ball, Diamond Ore will give you 1 diamond with a 25% chance of an additional diamond, or an average of 1.25 diamonds per ore.
+
+Adding a Dark Steel Ball will give you  a 150% chance of the main diamond, or 1.5 diamonds per ore..  The secondary diamond ends up being 25% * 150% * 200% = 75%, or 0.75 diamonds per ore.  Combining this with the primary diamond chance, a SAG Mill with Dark Steel Balls will, on average, get 2 diamonds per ore.
```
1.5 + 0.75 = 2?

VT-14 replied:
What's wrong with that?  Are you now going to tell me that 2 + 2 =/= 5? (sarcasm tag not showing).

HenryLoenwind commented on Alloy-Smelter.md:
```diff
@@ -1,5 +1,13 @@
+| Double-Layer        |   60 |     300,000 |
+| Octadic             |  100 |     500,000 |
+| Good Loot Cap (4)   |  160 |     800,000 |
+| Theoretical Max (5) |  200 |   1,000,000 |
```
Can you add a text about the speed being directly derived from the RF/T?

HenryLoenwind commented on Photovoltaic-Cells.md:
```diff
@@ -1,3 +1,25 @@
+Photovoltaic Cells generate power from sunlight.  They can connect horzontally as a free-form multiblock, even if using different tiers.  The ammount of power they generate vary throughout the day, maxing out around noon.  They must have a clear line of sight to the sky.
```
Please add that different tiers connecting is a config value, it defaults to true for 1.10, but is false for 1.7 and 1.11.

HenryLoenwind reviewed (changes requested):
Good work, Just a couple of details

VT-14 committed:
VT Update 1 Corrections

Alloy Smelter: Mentions that [processing] speed is directly related to RF/t.  Also mention the GUI button to lock the machine to either alloys or furnace modes.

* Photovoltaic Cells: Moved note about Cells connecting to different tiers and noted the settings in various versions.  Might want to just pull that paragraph out.

* SAG Mill: Major rewrite.  What math derp?  There was a math derp?

VT-14 commented:
Changes have been made.  Is that better?
